### PR TITLE
[Oluwalana Ajayi] - Sage  - Exploring Celo in the Gaming Industry for In-Game Assets and P2P Trading

### PR DESCRIPTION
**Pull request by lanacreates**
Thanks for submitting this tutorial for Celo Sage
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

 My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
 - [x] The title of the Pull Request matches the title of my tutorial.
 - [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x]  I've thoroughly explained the code, with inline code comments and text explanations.
 - [x] I've run a linter & prettifier to format my code blocks.
 - [x] I've run the tutorial to test it and the code works.

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -957,6 +957,13 @@ const Users: User[] = [
     website: 'blog/tutorials/truffle-and-celo-the-ultimate-guide-to-deploy-celo-dapps-with-truffle',
     tags: ['beginner','foundation', 'truffle'],
   },
+  {
+    title: 'Exploring Celo in the Gaming Industry for In-Game Assets and P2P Trading',
+    description: 'Learn how Celo is utilized in the Gaming Industry for In-Game Assets and P2P Trading',
+    preview: require('./showcase/beginner/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.png'),
```
Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:963]: There is some error coming due to this, check the name of the image and the location of image, make sure it is correct

File: blog/02-16-2023-Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  - name: Oluwalana Ajayi
+    title: Web3 Content Writer
+    url: https://github.com/lanacreates
+    image_url: https://github.com/lanacreates.png
+tags: [celosage, celowallet, celo, cusd, ledger]
+hide_table_of_contents: true
+slug: /tutorials/exploring-celo-in-the-gaming-industry-for-in-game-assets-and-p2p-trading
+---
+
+![header](../src/data-tutorials/showcase/intermediate/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.png)
```
Comment (Ishanpathak1) [inline blog/02-16-2023-Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.md:14]: Either the name or the location of the image is not correct as this is failing the build

**Review by Ishanpathak1 — COMMENTED** [review]
I think there is issue in location but in users.tsx it is showing different location from the metadata of .md file

In metadata it is ../src/data-tutorials/showcase/intermediate
and
In users.tsx it is ./showcase/beginner
Make the changes and let me know , after this we will be moving to publish section, thank you

**Comment by lanacreates** [comment]
> I think there is issue in location but in users.tsx it is showing different location from the metadata of .md file
> 
> In metadata it is ../src/data-tutorials/showcase/intermediate and In users.tsx it is ./showcase/beginner Make the changes and let me know , after this we will be moving to publish section, thank you

Hi It has been resolved

File: src/data-tutorials/users.tsx (hunk 1 above)
Comment (lanacreates) [reply src/data-tutorials/users.tsx:963]: Hi, the image now reflects above

Comment (Ishanpathak1) [inline src/data-tutorials/users.tsx:963]: change it to small case

Comment (lanacreates) [reply src/data-tutorials/users.tsx]: I just edited this, i think this might have been the issue. Thanks again

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**